src/parsers/variable.cr

```diff
@@ -41,6 +41,26 @@ module Mint
       end
     end
 
+    def variable_constant! : Ast::Variable
```

Comment:
Shouldn't this be called?

```suggestion
    def constant! : Ast::Variable
```

Reply:
I prefixed it with `variable_` to be consistent with some of the other methods in this file (i.e. `variable_attribute_name`, `variable_with_dashes`) but I do prefer your suggestion.

Happy to change it if you like?

Reply:
Some cases are legitimate like `variable_with_dashes`, yet in others (like this one, or `variable_attribute_name`) the `variable_` prefix seems like an implementation detail, and thus redundant, wdyt @gdotdesign ?

Reply:
We have `constant` which returns `Ast::Constant?` so by convention if it would be called `constant!` it should return `Ast::Constant` and raise when it could not parse, so I guess the prefix is fine.

Reply:
Oh, I see... In fact it's a constant _name_ in the end?

Reply:
Yes.